src/Form/matching-configuration.js

```diff
@@ -222,7 +222,11 @@ const matchingConfiguration = {
                 // user
                 email: css.__secret_do_not_use.email,
                 password: css.__secret_do_not_use.password,
-                username: css.__secret_do_not_use.username,
+                username: [
+                    css.__secret_do_not_use.username,
+                    // fix for `aa.com`
+                    'input[name="loginId"]'
+                ],
```

Comment:
Why are we adding it here instead of right in the css selectors?

Reply:
I was adding it here whilst I altered the types to allow arrays, I can move it back 👍🏻